static/images/logo_world_driven.svg

```diff
@@ -25,6 +25,6 @@
3 51.8638L265.478 51.8727L266.539 51.212ZM254.631 51.368L253.574 50.7005L253.571 50.7055L254.631 51.368ZM254.631 66.968L253.588 67.6568C253.591 67.6614 253.594 67.6659 253.597 67.6704L254.631 66.968ZM259.519 72.27C256.376 72.27 253.961 71.2032 252.142 69.1118L250.256 70.7522C252.597 73.4448 255.729 74.77 259.519 74.77V72.27ZM252.148 69.1185C250.339 67.0076 249.329 63.745 249.329 59.116H246.829C246.829 64.055 247.9 68.0031 250.25 70.7455L252.148 69.1185ZM249.329 59.116C249.329 54.487 250.339 51.2244 252.148 49.1135L250.25 47.4865C247.9 50.2289 246.829 54.177 246.829 59.116H249.329ZM252.147 49.1146C254.007 46.9504 256.441 45.858 259.571 45.858V43.358C255.768 43.358 252.621 44.7269 250.251 47.4854L252.147 49.1146ZM259.571 45.858C261.576 45.858 263.087 46.2312 264.188 46.8807L265.458 44.7273C263.856 43.7821 261.865 43.358 259.571 43.358V45.858ZM264.2 46.8876C265.458 47.6111 266.452 48.4748 267.207 49.4708L269.2 47.9612C268.221 46.6692 266.962 45.5916 265.446 44.7204L264.2 46.8876ZM268.203 49.966H268.515V47.466H268.203V49.966ZM269.751 48.5259C269.688 48.1211 269.622 47.4986 269.553 46.6403L267.061 46.8397C267.131 47.7148 267.203 48.4096 267.28 48.9061L269.751 48.5259ZM269.554 46.6441C269.485 45.7552 269.453 45.0811 269.453 44.608H266.953C266.953 45.1749 266.991 45.9222 267.061 46.8359L269.554 46.6441ZM269.453 44.608V33.48H266.953V44.608H269.453ZM268.203 34.73H272.779V32.23H268.203V34.73ZM271.529 33.48V73H274.029V33.48H271.529ZM272.779 71.75H269.087V74.25H272.779V71.75ZM270.317 72.7779L269.641 69.0339L267.181 69.4781L267.857 73.2221L270.317 72.7779ZM268.411 68.006H268.203V70.506H268.411V68.006ZM267.182 68.5351C266.462 69.5555 265.488 70.4467 264.232 71.2001L265.518 73.3439C267.036 72.4333 268.281 71.3138 269.224 69.9769L267.182 68.5351ZM264.232 71.2001C263.096 71.8816 261.553 72.27 259.519 72.27V74.77C261.853 74.77 263.881 74.3264 265.518 73.3439L264.232 71.2001ZM260.247 70.974C263.389 70.974 265.928 70.1215 267.44 68.075L265.43 66.589C264.585 67.7319 262.998 68.474 260.247 68.474V70.974ZM267.438 68.0786C268.883 66.138 269.505 63.3775 269.505 60H267.005C267.005 63.1399 266.414 65.2673 265.433 66.5854L267.438 68.0786ZM269.505 60V59.168H267.005V60H269.505ZM269.505 59.168C269.505 55.5953 268.921 52.6728 267.6 50.5513L265.478 51.8727C266.445 53.4259 267.005 55.8073 267.005 59.168H269.505ZM267.606 50.5602C266.149 48.1758 263.521 47.154 260.195 47.154V49.654C263.04 49.654 264.642 50.5042 265.473 51.8638L267.606 50.5602ZM260.195 47.154C257.333 47.154 255.066 48.3392 253.574 50.7005L255.688 52.0355C256.693 50.4448 258.135 49.654 260.195 49.654V47.154ZM253.571 50.7055C252.191 52.9144 251.561 55.7897 251.561 59.22H254.061C254.061 56.0636 254.645 53.7042 255.691 52.0305L253.571 50.7055ZM251.561 59.22C251.561 62.6758 252.187 65.5355 253.588 67.6568L255.674 66.2792C254.649 64.7259 254.061 62.4202 254.061 59.22H251.561ZM253.597 67.6704C255.11 69.897 257.397 70.974 260.247 70.974V68.474C258.105 68.474 256.649 67.7137 255.665 66.2656L253.597 67.6704Z" fill="#3C6A82"/>
```

Comment:
Same here, the logos itself doesn't need to be touched. They were provided recently together with the color palette https://github.com/TooAngel/worlddriven/pull/158